src/App.tsx

```diff
@@ -1,73 +1,32 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { Autocomplete } from './components/Autocomplete';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+
+  const handleSelectedPerson = (person: Person) => {
+    setSelectedPerson(person);
+  };
```

Comment:
The `handleSelectedPerson` function should be updated to handle `null` values as well, since the `Autocomplete` component's `onSelected` prop expects a `Person | null` type. Consider updating the function signature to `const handleSelectedPerson = (person: Person | null) => { ... }`.